.github/contributors.json

```diff
@@ -108,6 +108,11 @@
     "github_login": "alb3rto",
     "twitter_username": ""
   },
+  {
+    "name": "Aleksey Storin",
+    "github_login": "ghostforpy",
+    "twitter_username": ""
+  },
```

Comment:
This might be autogenerated so you may need to remove these changes in this file and the other CONTRIBUTORS.md file. Not sure how to add one manually or if it's allowed

Reply:
Deleted